tests/providers/neo4j/hooks/test_neo4j.py

```diff
@@ -15,51 +15,94 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-import json
 import unittest
 from unittest import mock
 
+from parameterized import parameterized
+
 from airflow.models import Connection
 from airflow.providers.neo4j.hooks.neo4j import Neo4jHook
 
 
 class TestNeo4jHookConn(unittest.TestCase):
-    def setUp(self):
-        super().setUp()
-        self.neo4j_hook = Neo4jHook()
-        self.connection = Connection(
-            conn_type='neo4j', login='login', password='password', host='host', schema='schema'
+    @parameterized.expand(
+        [
+            [dict(), "bolt://host:7687"],
```

Comment:
```suggestion
            [{}, "bolt://host:7687"],
```

Reply:
I prefer to avoid initializing an empty dictionary with this syntax because it is ambiguous. I always have to wonder if this will make an empty dictionary(``{}==dictt()``) or a set(``{}==set()``).

Reply:
explicit is better than implicit :)